docs/operators/remove.md

```diff
@@ -0,0 +1,343 @@
+
+Example usage:
+
+<hr>
+
+Remove value from body
+```yaml
+- type: remove 
+    field: 
+    - key1
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": {
+    "key1": "val1",
+  }
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": { }
+}
+```
+
+</td>
+</tr>
+</table>
+
+<hr>
+
+Remove object from body
+```yaml
+- type: remove 
+    field: 
+    - object
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": {
+    "object": {
+      "nestedkey": "nestedval"
+    }
+  },
+  "key": "val"
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": { 
+     "key": "val"
+  }
+}
+```
+
+</td>
+</tr>
+</table>
+
+<hr>
+
+Remove multiple Values from body
+```yaml
+- type: remove 
+    field: 
+    - object
+    - key
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": {
+    "object": {
+      "nestedkey": "nestedval"
+    },
+    "key": "val"
+  },
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": { }
+}
+```
+
+</td>
+</tr>
+</table>
+
+<hr>
+
+Remove Value from attributes
+```yaml
+- type: remove 
+    field: 
+    - $attributes.otherkey
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { 
+    "otherkey": "val"
+  },  
+  "body": {
+    "object": {
+      "nestedkey": "nestedval"
+    }, 
+    "key": "val"
+  },
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": {  },  
+  "body": { 
+    "object": {
+      "nestedkey": "nestedval"
+    }, 
+    "key": "val"
+  }
+}
+```
+
+</td>
+</tr>
+</table>
+
+<hr>
+
+Remove multiple values from attributes
+```yaml
+- type: remove 
+    field: 
+    - $attributes.otherkey
+    - $attributes.otherkey2
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { 
+    "otherkey": "val",
+    "otherkey2": "val2"
+  },  
+  "body": {
+    "object": {
+      "nestedkey": "nestedval"
+    }, 
+    "key": "val"
+  },
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": {  },  
+  "body": { 
+    "object": {
+      "nestedkey": "nestedval"
+    }, 
+    "key": "val"
+  }
+}
+```
+
+</td>
+</tr>
+</table>
+
+<hr>
+
+Remove Value from resource
+```yaml
+- type: remove 
+    field: 
+    - $resource.otherkey
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { 
+    "otherkey": "val"
+  },
+  "attributes": {  },  
+  "body": {
+    "object": {
+      "nestedkey": "nestedval"
+    }, 
+    "key": "val"
+  },
```

Comment:
Same point as above. Keep the body simple since it otherwise draws focus away from the example